r2t2/core.py

```diff
@@ -25,7 +25,8 @@ def add_record(out, record):
             index = 1
             out += f"Referenced in: {record.name}"
             out += f"\nSource file: {record.source}"
-            out += f"\nLine: {record.line}\n"
+            if record.line is not None:
+                out += f"\nLine: {record.line}\n"
```

Comment:
This messes up the output for parsing notebooks on my system (screenshot attached).

I think the way to fix it is to move the first `\n` on line 29 to the end of line 27 (I'm happy to push this, not sure if I have access):
```
27: out += f"\nSource file: {record.source}\n"
28: if record.line is not None:
29:    out += f"Line: {record.line}\n"
```
![Selection_017](https://user-images.githubusercontent.com/6338509/92165838-ad6bed00-ee05-11ea-8e80-c360e634b0fd.png)

Reply:
With that small change, it looks great!

Reply:
Thanks, just made that tweak!

Reply:
Looks good!